ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -97,17 +99,25 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
     };
   }
 
-  createAccount(): void {
+  async createAccount() {
     const {invitationKey, setProfile} = this.props;
     this.setState({creatingAccount: true});
-    profileApi().createAccount({profile: this.profileObj, invitationKey: invitationKey})
-      .then((savedProfile) => {
-        this.setState({profile: savedProfile, creatingAccount: false});
-        setProfile(savedProfile, {stepName: 'accountCreationSuccess', backgroundImages: signedOutImages.login});
-      }).catch(error => {
-        console.log(error);
-        this.setState({creatingAccount: false});
-      });
+    try {
+      await profileApi().createAccount({profile: this.profileObj, invitationKey: invitationKey})
+        .then((savedProfile) => {
+          this.setState({profile: savedProfile, creatingAccount: false});
+          setProfile(savedProfile, {stepName: 'accountCreationSuccess', backgroundImages: signedOutImages.login});
+        });
+    } catch (e) {
```

Comment:
Can we use this here instead of rehandling the exception separately?
https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/retry.tsx#L34